code/modules/vehicles/mecha/working/clarke.dm

```diff
@@ -1,11 +1,11 @@
-///Lavaproof, fireproof, fast mech with low armor and higher energy consumption, cannot strafe and has an internal ore box.
+///Lavaproof, fireproof, fast mech with low armor and higher energy consumption and has an internal ore box.
 /obj/vehicle/sealed/mecha/clarke
 	desc = "Combining man and machine for a better, stronger engineer. Can even resist lava!"
```

Comment:
honestly at this point i would change the description cause it aint an engi mech but thats a nitpick